### PR TITLE
Issue #1623 Include link in README to the build process

### LINKED ISSUE #1623: Include link in README to the build process
The README doesn't say anything about building minishift from source, but it will be useful to have.

### PR DESCRIPTION
**Pull request by jaseemabid**
Fixes #1623

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -48,6 +48,7 @@ using, and contributing to Minishift:
 - https://docs.openshift.org/latest/minishift/getting-started/index.html[Getting started with Minishift]
 - https://docs.openshift.org/latest/minishift/using/index.html[Using Minishift]
 - https://docs.openshift.org/latest/minishift/contributing/index.html[Contributing to Minishift]
+- https://docs.openshift.org/latest/minishift/contributing/developing.html[Developing and building Minishift locally]
```
Comment (gbraad) [inline README.adoc]:
```diff
- - https://docs.openshift.org/latest/minishift/contributing/index.html[Contributing to Minishift]
+ - https://docs.openshift.org/latest/minishift/contributing/index.html[Developing and contributing to Minishift]
```

Comment (hferentschik) [reply README.adoc]: +1 In this case, we just update the link title, instead of adding a whole new link

**Comment by LalatenduMohanty** [comment]
ok to test

**Comment by LalatenduMohanty** [comment]
@jaseemabid Do you want to fix your PR and merge it. Let us know if you are planning to fix it.

**Comment by hferentschik** [comment]
Merged via pull request #1642

**Pull request closed without merging**